docs/examples/FixedOptions.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+import Select, { components } from '../../src';
+import { colourOptions } from '../data';
+
+type State = {
+  value: [{ [string]: string }]
+};
+
+const TagStyle = {
```

Comment:
Could this just be `styles` or similar? Pascal case is typically reserved for classes/types within JS.